<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newTournament</class>
 <widget class="QDialog" name="newTournament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tableWidgetPlayers">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>44</y>
      <width>651</width>
      <height>401</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>771</width>
      <height>471</height>
     </size>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>1</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="buttonImport">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>6</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Importar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonEmparcerar">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>6</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Emparcerar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDelete">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>6</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Remover</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPrev">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>8</y>
       <width>26</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonNext">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>8</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRound">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>14</y>
       <width>57</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Round:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonQuit">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>6</y>
       <width>96</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Fechar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonWhiteWins">
     <property name="geometry">
      <rect>
       <x>419</x>
       <y>8</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1-0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDraws">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>8</y>
       <width>48</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>½ - ½</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBlackWins">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>8</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0-1</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
